ldk/javascript/src/whisper/types.ts

```diff
@@ -444,6 +452,12 @@ export type RichTextEditor = WhisperComponent<WhisperComponentType.RichTextEdito
 
 export type Divider = WhisperComponent<WhisperComponentType.Divider>;
 
+export type Progress = WhisperComponent<WhisperComponentType.Progress> & {
+  determinate?: number;
+  shape?: ProgressShape;
+  size?: StyleSize;
```

Comment:
My last question is about the choice to use `StyleSize` here. `none` is an option for that type, so what happens if someone uses that as a value?